mbedtls-sys/build/config.rs

```diff
@@ -382,7 +388,7 @@ const DEFAULT_DEFINES: &'static [CDefine] = &[
     ("MBEDTLS_SSL_DEFAULT_TICKET_LIFETIME",               Undefined), // default: 86400
     ("MBEDTLS_PSK_MAX_LEN",                               Undefined), // default: 32
     ("MBEDTLS_SSL_COOKIE_TIMEOUT",                        Undefined), // default: 60
-    ("MBEDTLS_SSL_CIPHERSUITES",                          Undefined), // no default
```

Comment:
I think I mentioned this before, the default here refers to what you get when you don't define it. That's different from what's shown in the commented-out define in the config.h source code.